src/main/views/citizen-hub.njk

```diff
@@ -0,0 +1,116 @@
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h2 class="govuk-heading-l">
+        {{ header + userCase.firstName + ' ' + userCase.lastName }} vs {{ userCase.respondents[0].respondentName }}
+      </h2>
+      <p id="caseNumber" class="govuk-body">{{ caseNumber + userCase.ethosCaseReference }}</p>
+
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds-from-desktop">
+          <div class="top-border-blue">
+            {{ hmctsProgressBar({
+            items: [{
+              label:
+              {
+                text: accepted
+              },
+              complete: currentState.isAfter('SUBMITTED'),
+              active: currentState.isRightAfter('ACCEPTED')
+            }, {
+              label: {
+                text: received
+              },
+              complete: currentState.isAfter('ACCEPTED'),
+              active: currentState.isRightAfter('RESPONSE_RECEIVED')
+            }, {
+              label: {
+                text: details
+              },
+              complete: currentState.isAfter('RESPONSE_RECEIVED'),
+              active: currentState.isRightAfter('HEARING_DETAILS')
+            }, {
+              label: {
+                text: decision
+              },
+              complete: currentState.isAfter('HEARING_DETAILS'),
+              active: currentState.isRightAfter('CLAIM_DECISION')
+             }],
+              classes: ''
+            }) }}
+
+            <ol class="app-task-list">
+              {% for section in sections %}
+                <li>
+                  <h2 class="app-task-list__section">
+                    {{getContent(section.title)}}
+                  </h2>
+                  <ul class="app-task-list__items">
+                    {% for link in section.links %}
+                      <li class="app-task-list__item">
+                        <span class="app-task-list__task-name">
+                          {% if getContent(link.status) === "Not available yet" %}
+                            {{ getContent(link.linkTxt) }}
+                          {% else %}
+                            <a href="{{ getContent(link.url) }}" class="govuk-link">
+                              {{ getContent(link.linkTxt) }}
+                            </a>
+                          {% endif %}
+                        </span>
+
+                        {{ govukTag({
+                      text: getContent(link.status),
+                      classes: "app-task-list__tag govuk-tag" + getContent(link.statusColor)
+                    }) }}
+                      </li>
+                    {% endfor %}
+                  </ul>
+                </li>
+              {% endfor %}
+            </ol>
+          </div>
+        </div>
+        <div class="govuk-grid-column-one-third-from-desktop">
+          <div class="top-border-blue">
+            <h2 class="govuk-heading-s">
+              I want to...
```

Comment:
Just a small thing - would it be better to move the English in the citizen-hub.njk into a translation files?